fastparquet/writer.py

```diff
@@ -137,7 +140,10 @@ def convert(data, se):
         out = data.values
     elif dtype == "O":
         if converted_type == parquet_thrift.ConvertedType.UTF8:
-            out = array_encode_utf8(data)
+            if PY2:
+                out = np.array([x.encode('utf8') for x in data], dtype="O")
```

Comment:
This does not appear to be true:
```fastparquet.speedups.array_encode_utf8(np.array([u'ef¬∆∫˚'], dtype="O"))
array(['ef\xc2\xac\xe2\x88\x86\xe2\x88\xab\xcb\x9a'], dtype=object)```
I suspect, again, that the difference is explicitly labeling the input as unicode. Without that, it would be the equivalent of py3's bytes.